### PR TITLE
added more clarity to slack and hipchat configurations

### PR DESCRIPTION
**Pull request by abhijitkini**

### CODE REVIEW

File: sources/ci_configure.md [hunk 1]
```diff
@@ -1089,7 +1090,7 @@ integrations:
       on_success: never
       on_failure: always
 ```
-* `integrationName` value is the name of the account integration you added to the 'Subscription' settings.
+* `integrationName` value is the name of the Slack integration you added to the 'Subscription' settings in Step 1. **IMPORTANT** The names in the 'Subscription' 'Integration' settings in Step 1 and the `shippable.yml` have to be exactly the same. If they are not, the build will fail with error as [described here](ci_troubleshoot/#integration-name-specified-in-yml-does-not-match).
```
Comment (a-murphy) [inline sources/ci_configure.md]: ... fail with an error as ...

Could you add "an" before "error"?  Or "the error described here"?  It's a specific error and there will only be one error causing the failure.

File: sources/ci_configure.md [hunk 2]
```diff
@@ -1138,7 +1140,7 @@ integrations:
       on_success: never
       on_failure: always
 ```
-* `integrationName` value is the name of the account integration you added to 'Subscription' settings.
+* `integrationName` value is the name of the HipChat integration you added to the 'Subscription' settings in Step 1. **IMPORTANT** The names in the 'Subscription' 'Integration' settings in Step 1 and the `shippable.yml` have to be exactly the same. If they are not, the build will fail with error as [described here](ci_troubleshoot/#integration-name-specified-in-yml-does-not-match).
```
Comment (a-murphy) [inline sources/ci_configure.md]: And the same comment as above here.

File: sources/ci_configure.md (hunk 1 above)
Comment (abhijitkini) [inline sources/ci_configure.md]: Added "an" before "error"

File: sources/ci_configure.md (hunk 2 above)
Comment (abhijitkini) [inline sources/ci_configure.md]: Added "an" before "error"

**Pull request merged**